### ISSUE TITLE
Installation on Windows (Powershell) fails with File Not Found

### ISSUE THREAD

**Issue by thclark** [octue/octue-sdk-python#106]
### What is the current behavior?

Installation error when trying to install `octue` on windows machines. 

### What is the expected behavior?

`pip install octue` should work correctly.

### Your environment

  - Library Version: `x.y.z`
  - Platform Windows (Powershell)

### Other information

Please give as much detail as you can, like:

- [x] stacktraces

```
Installing collected packages: octue, nodeenv, keyring, identify, fs, crc32c, cfgv, wheel, twine, tox, pre-commit, gcp-storage-emulator, flake8, data-gateway, coverage, black
    Running setup.py install for octue ... error
    ERROR: Command errored out with exit status 1:
     command: 'C:\Users\TheUser\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\TheUser\\AppData\\Local\\Temp\\pip-install-1nx0mni5\\octue_a5977395d6064c29ad56d7bd75216121\\setup.py'"'"'; __file__='"'"'C:\\Users\\TheUser\\AppData\\Local\\Temp\\pip-install-1nx0mni5\\octue_a5977395d6064c29ad56d7bd75216121\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\TheUser\AppData\Local\Temp\pip-record-fhljtv2y\install-record.txt' --single-version-externally-managed --user --prefix= --compile --install-headers 'C:\Users\TheUser\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\Include\octue'
         cwd: C:\Users\TheUser\AppData\Local\Temp\pip-install-1nx0mni5\octue_a5977395d6064c29ad56d7bd75216121\
    Complete output (182 lines):
    running install
    running build
    running build_py
    file cli.py (for module cli) not found
    creating build
    creating build\lib
    creating build\lib\octue
    copying octue\cli.py -> build\lib\octue
    copying octue\definitions.py -> build\lib\octue
    copying octue\exceptions.py -> build\lib\octue
    copying octue\logging_handlers.py -> build\lib\octue
    copying octue\runner.py -> build\lib\octue
    copying octue\__init__.py -> build\lib\octue
    creating build\lib\octue\mixins
    copying octue\mixins\base.py -> build\lib\octue\mixins
    copying octue\mixins\cool_nameable.py -> build\lib\octue\mixins
    copying octue\mixins\filterable.py -> build\lib\octue\mixins
    copying octue\mixins\hashable.py -> build\lib\octue\mixins
   copying octue\mixins\identifiable.py -> build\lib\octue\mixins
    copying octue\mixins\loggable.py -> build\lib\octue\mixins
    copying octue\mixins\pathable.py -> build\lib\octue\mixins
    copying octue\mixins\serialisable.py -> build\lib\octue\mixins
    copying octue\mixins\taggable.py -> build\lib\octue\mixins
    copying octue\mixins\__init__.py -> build\lib\octue\mixins
    creating build\lib\octue\resources
    copying octue\resources\analysis.py -> build\lib\octue\resources
    copying octue\resources\child.py -> build\lib\octue\resources
    copying octue\resources\datafile.py -> build\lib\octue\resources
    copying octue\resources\dataset.py -> build\lib\octue\resources
    copying octue\resources\filter_containers.py -> build\lib\octue\resources
    copying octue\resources\manifest.py -> build\lib\octue\resources
    copying octue\resources\tag.py -> build\lib\octue\resources
    copying octue\resources\__init__.py -> build\lib\octue\resources
    creating build\lib\octue\templates
    copying octue\templates\templates.py -> build\lib\octue\templates
    copying octue\templates\__init__.py -> build\lib\octue\templates
    creating build\lib\octue\utils
    copying octue\utils\decoders.py -> build\lib\octue\utils
    copying octue\utils\encoders.py -> build\lib\octue\utils
    copying octue\utils\folders.py -> build\lib\octue\utils
    copying octue\utils\gen_uuid.py -> build\lib\octue\utils
    copying octue\utils\isfile.py -> build\lib\octue\utils
    copying octue\utils\isfolder.py -> build\lib\octue\utils
    copying octue\utils\persistence.py -> build\lib\octue\utils
    copying octue\utils\processes.py -> build\lib\octue\utils
    copying octue\utils\__init__.py -> build\lib\octue\utils
    creating build\lib\octue\resources\communication
    copying octue\resources\communication\service_backends.py -> build\lib\octue\resources\communication
    copying octue\resources\communication\__init__.py -> build\lib\octue\resources\communication
    creating build\lib\octue\resources\communication\google_pub_sub
    copying octue\resources\communication\google_pub_sub\service.py -> build\lib\octue\resources\communication\google_pub_sub
    copying octue\resources\communication\google_pub_sub\subscription.py -> build\lib\octue\resources\communication\google_pub_sub
    copying octue\resources\communication\google_pub_sub\topic.py -> build\lib\octue\resources\communication\google_pub_sub
    copying octue\resources\communication\google_pub_sub\__init__.py -> build\lib\octue\resources\communication\google_pub_sub
    creating build\lib\octue\utils\cloud
    copying octue\utils\cloud\credentials.py -> build\lib\octue\utils\cloud
    copying octue\utils\cloud\emulators.py -> build\lib\octue\utils\cloud
    copying octue\utils\cloud\__init__.py -> build\lib\octue\utils\cloud
    creating build\lib\octue\utils\cloud\storage
    copying octue\utils\cloud\storage\client.py -> build\lib\octue\utils\cloud\storage
    copying octue\utils\cloud\storage\path.py -> build\lib\octue\utils\cloud\storage
    copying octue\utils\cloud\storage\__init__.py -> build\lib\octue\utils\cloud\storage
    creating build\lib\tests
    creating build\lib\tests\mixins
    copying tests\mixins\test_base.py -> build\lib\tests\mixins
    copying tests\mixins\test_cool_nameable.py -> build\lib\tests\mixins
    copying tests\mixins\test_filterable.py -> build\lib\tests\mixins
    copying tests\mixins\test_hashable.py -> build\lib\tests\mixins
    copying tests\mixins\test_identifiable.py -> build\lib\tests\mixins
    copying tests\mixins\test_loggable.py -> build\lib\tests\mixins
    copying tests\mixins\test_pathable.py -> build\lib\tests\mixins
    copying tests\mixins\test_serialisable.py -> build\lib\tests\mixins
    copying tests\mixins\test_taggable.py -> build\lib\tests\mixins
    copying tests\mixins\__init__.py -> build\lib\tests\mixins
    creating build\lib\tests\resources
    copying tests\resources\test_analysis.py -> build\lib\tests\resources
    copying tests\resources\test_datafile.py -> build\lib\tests\resources
    copying tests\resources\test_dataset.py -> build\lib\tests\resources
    copying tests\resources\test_filter_containers.py -> build\lib\tests\resources
    copying tests\resources\test_manifest.py -> build\lib\tests\resources
    copying tests\resources\test_tag.py -> build\lib\tests\resources
    copying tests\resources\__init__.py -> build\lib\tests\resources
    creating build\lib\tests\templates
    copying tests\templates\test_templates.py -> build\lib\tests\templates
    copying tests\templates\test_template_apps.py -> build\lib\tests\templates
    copying tests\templates\__init__.py -> build\lib\tests\templates
    creating build\lib\tests\utils
    copying tests\utils\test_persistence.py -> build\lib\tests\utils
    copying tests\utils\__init__.py -> build\lib\tests\utils
    creating build\lib\tests\resources\communication
    copying tests\resources\communication\test_service_backends.py -> build\lib\tests\resources\communication
    copying tests\resources\communication\__init__.py -> build\lib\tests\resources\communication
    creating build\lib\tests\resources\communication\google_pub_sub
    copying tests\resources\communication\google_pub_sub\test_service.py -> build\lib\tests\resources\communication\google_pub_sub
    copying tests\resources\communication\google_pub_sub\__init__.py -> build\lib\tests\resources\communication\google_pub_sub
    creating build\lib\tests\utils\cloud
    copying tests\utils\cloud\test_credentials.py -> build\lib\tests\utils\cloud
    copying tests\utils\cloud\__init__.py -> build\lib\tests\utils\cloud
    creating build\lib\tests\utils\cloud\storage
    copying tests\utils\cloud\storage\test_client.py -> build\lib\tests\utils\cloud\storage
    copying tests\utils\cloud\storage\test_path.py -> build\lib\tests\utils\cloud\storage
    copying tests\utils\cloud\storage\__init__.py -> build\lib\tests\utils\cloud\storage
    running egg_info
    creating octue.egg-info
    writing octue.egg-info\PKG-INFO
    writing dependency_links to octue.egg-info\dependency_links.txt
    writing entry points to octue.egg-info\entry_points.txt
    writing requirements to octue.egg-info\requires.txt
    writing top-level names to octue.egg-info\top_level.txt
    writing manifest file 'octue.egg-info\SOURCES.txt'
    file cli.py (for module cli) not found
    reading manifest file 'octue.egg-info\SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    writing manifest file 'octue.egg-info\SOURCES.txt'
    creating build\lib\octue\templates\template-child-services
    creating build\lib\octue\templates\template-child-services\elevation_service
    copying octue\templates\template-child-services\elevation_service\.gitignore -> build\lib\octue\templates\template-child-services\elevation_service
    copying octue\templates\template-child-services\elevation_service\app.py -> build\lib\octue\templates\template-child-services\elevation_service
    copying octue\templates\template-child-services\elevation_service\requirements.txt -> build\lib\octue\templates\template-child-services\elevation_service
    copying octue\templates\template-child-services\elevation_service\setup.py -> build\lib\octue\templates\template-child-services\elevation_service
    copying octue\templates\template-child-services\elevation_service\twine.json -> build\lib\octue\templates\template-child-services\elevation_service
    creating build\lib\octue\templates\template-child-services\elevation_service\data
    creating build\lib\octue\templates\template-child-services\elevation_service\data\configuration
    copying octue\templates\template-child-services\elevation_service\data\configuration\values.json -> build\lib\octue\templates\template-child-services\elevation_service\data\configuration
    creating build\lib\octue\templates\template-child-services\parent_service
    copying octue\templates\template-child-services\parent_service\.gitignore -> build\lib\octue\templates\template-child-services\parent_service
    copying octue\templates\template-child-services\parent_service\app.py -> build\lib\octue\templates\template-child-services\parent_service
    copying octue\templates\template-child-services\parent_service\requirements.txt -> build\lib\octue\templates\template-child-services\parent_service
    copying octue\templates\template-child-services\parent_service\setup.py -> build\lib\octue\templates\template-child-services\parent_service
    copying octue\templates\template-child-services\parent_service\twine.json -> build\lib\octue\templates\template-child-services\parent_service
    creating build\lib\octue\templates\template-child-services\parent_service\data
    creating build\lib\octue\templates\template-child-services\parent_service\data\configuration
    copying octue\templates\template-child-services\parent_service\data\configuration\children.json -> build\lib\octue\templates\template-child-services\parent_service\data\configuration
    creating build\lib\octue\templates\template-child-services\parent_service\data\input
    copying octue\templates\template-child-services\parent_service\data\input\values.json -> build\lib\octue\templates\template-child-services\parent_service\data\input
    creating build\lib\octue\templates\template-child-services\wind_speed_service
    copying octue\templates\template-child-services\wind_speed_service\.gitignore -> build\lib\octue\templates\template-child-services\wind_speed_service
    copying octue\templates\template-child-services\wind_speed_service\app.py -> build\lib\octue\templates\template-child-services\wind_speed_service
    copying octue\templates\template-child-services\wind_speed_service\requirements.txt -> build\lib\octue\templates\template-child-services\wind_speed_service
    copying octue\templates\template-child-services\wind_speed_service\setup.py -> build\lib\octue\templates\template-child-services\wind_speed_service
    copying octue\templates\template-child-services\wind_speed_service\twine.json -> build\lib\octue\templates\template-child-services\wind_speed_service
    creating build\lib\octue\templates\template-child-services\wind_speed_service\data
    creating build\lib\octue\templates\template-child-services\wind_speed_service\data\configuration
    copying octue\templates\template-child-services\wind_speed_service\data\configuration\values.json -> build\lib\octue\templates\template-child-services\wind_speed_service\data\configuration
    creating build\lib\octue\templates\template-python-fractal
    copying octue\templates\template-python-fractal\.gitignore -> build\lib\octue\templates\template-python-fractal
    copying octue\templates\template-python-fractal\app.py -> build\lib\octue\templates\template-python-fractal
    copying octue\templates\template-python-fractal\requirements.txt -> build\lib\octue\templates\template-python-fractal
    copying octue\templates\template-python-fractal\setup.py -> build\lib\octue\templates\template-python-fractal
    copying octue\templates\template-python-fractal\twine.json -> build\lib\octue\templates\template-python-fractal
    creating build\lib\octue\templates\template-python-fractal\data
    creating build\lib\octue\templates\template-python-fractal\data\configuration
    copying octue\templates\template-python-fractal\data\configuration\configuration_values.json -> build\lib\octue\templates\template-python-fractal\data\configuration
    creating build\lib\octue\templates\template-python-fractal\fractal
    copying octue\templates\template-python-fractal\fractal\__init__.py -> build\lib\octue\templates\template-python-fractal\fractal
    copying octue\templates\template-python-fractal\fractal\fractal.py -> build\lib\octue\templates\template-python-fractal\fractal
    copying octue\templates\template-python-fractal\fractal\mandelbrot.py -> build\lib\octue\templates\template-python-fractal\fractal
    creating build\lib\octue\templates\template-using-manifests
    copying octue\templates\template-using-manifests\.gitignore -> build\lib\octue\templates\template-using-manifests
    copying octue\templates\template-using-manifests\app.py -> build\lib\octue\templates\template-using-manifests
    copying octue\templates\template-using-manifests\requirements.txt -> build\lib\octue\templates\template-using-manifests
    copying octue\templates\template-using-manifests\setup.py -> build\lib\octue\templates\template-using-manifests
    copying octue\templates\template-using-manifests\twine.json -> build\lib\octue\templates\template-using-manifests
    creating build\lib\octue\templates\template-using-manifests\cleaner
    copying octue\templates\template-using-manifests\cleaner\__init__.py -> build\lib\octue\templates\template-using-manifests\cleaner
    copying octue\templates\template-using-manifests\cleaner\clean.py -> build\lib\octue\templates\template-using-manifests\cleaner
    copying octue\templates\template-using-manifests\cleaner\read_csv_files.py -> build\lib\octue\templates\template-using-manifests\cleaner
    copying octue\templates\template-using-manifests\cleaner\read_dat_file.py -> build\lib\octue\templates\template-using-manifests\cleaner
    creating build\lib\octue\templates\template-using-manifests\data
    creating build\lib\octue\templates\template-using-manifests\data\configuration
    copying octue\templates\template-using-manifests\data\configuration\values.json -> build\lib\octue\templates\template-using-manifests\data\configuration
    creating build\lib\octue\templates\template-using-manifests\data\input
    copying octue\templates\template-using-manifests\data\input\manifest.json -> build\lib\octue\templates\template-using-manifests\data\input
    creating build\lib\octue\templates\template-using-manifests\data\input\raw_met_mast_data
    creating build\lib\octue\templates\template-using-manifests\data\input\raw_met_mast_data\08DEC
    copying octue\templates\template-using-manifests\data\input\raw_met_mast_data\08DEC\High Res Meteorological Mast Data - 8 Dec_1.csv -> build\lib\octue\templates\template-using-manifests\data\input\raw_met_mast_data\08DEC
    copying octue\templates\template-using-manifests\data\input\raw_met_mast_data\08DEC\High Res Meteorological Mast Data - 8 Dec_2.csv -> build\lib\octue\templates\template-using-manifests\data\input\raw_met_mast_data\08DEC
    copying octue\templates\template-using-manifests\data\input\raw_met_mast_data\08DEC\meta - 8 Dec.dat -> build\lib\octue\templates\template-using-manifests\data\input\raw_met_mast_data\08DEC
    file cli.py (for module cli) not found
    running install_lib
    copying build\lib\octue\templates\template-using-manifests\data\input\raw_met_mast_data\08DEC\High Res Meteorological Mast Data - 8 Dec_1.csv -> C:\Users\TheUser\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\octue\templates\template-using-manifests\data\input\raw_met_mast_data\08DEC
    error: could not create 'C:\Users\TheUser\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\octue\templates\template-using-manifests\data\input\raw_met_mast_data\08DEC\High Res Meteorological Mast Data - 8 Dec_1.csv': No such file or directory
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'C:\Users\TheUser\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\TheUser\\AppData\\Local\\Temp\\pip-install-1nx0mni5\\octue_a5977395d6064c29ad56d7bd75216121\\setup.py'"'"'; __file__='"'"'C:\\Users\\TheUser\\AppData\\Local\\Temp\\pip-install-1nx0mni5\\octue_a5977395d6064c29ad56d7bd75216121\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\TheUser\AppData\Local\Temp\pip-record-fhljtv2y\install-record.txt' --single-version-externally-managed --user --prefix= --compile --install-headers 'C:\Users\TheUser\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\Include\octue' Check the logs for full command output.
```

**Comment by thclark** [comment]
@cortadocodes this looks like this CSV file being missing from the MANIFEST file. The recursive-include statement doesn't work on non python files, (see the manifest in the twined library) so we probably need to include this.

**Comment by cortadocodes** [comment]
Also, do we need some kind of windows testing platform in addition to the CI we currently have?

**Comment by cortadocodes** [comment]
I have no idea of the quality of this source, but maybe using `setuptools_scm` rather than `MANIFEST.in` would solve the problem? - https://www.remarkablyrestrained.com/python-setuptools-manifest-in/

**Comment by cortadocodes** [comment]
Putting these here for reference as possible solutions:
* https://docs.python.org/3/distutils/setupscript.html#distutils-installing-package-data
* https://docs.python.org/3/distutils/setupscript.html#distutils-additional-files

**Comment by cortadocodes** [comment]
Fixed by 0.1.9
